# Cloud Info
Header header

# Point cloud messages
sensor_msgs/PointCloud2 cloud_raw  # original cloud deskewed
sensor_msgs/PointCloud2 cloud_corner    # extracted corner feature
sensor_msgs/PointCloud2 cloud_surface   # extracted surface feature
sensor_msgs/PointCloud2 cloud_corner_sharp    # extracted corner feature
sensor_msgs/PointCloud2 cloud_surface_flat   # extracted surface feature


